modules/apps/frontend-js/frontend-js-spa-web/src/main/resources/META-INF/resources/senna/surface/Surface.js

```diff
@@ -95,7 +93,7 @@ class Surface extends Disposable {
 	addContent(screenId, opt_content) {
 		var child = this.defaultChild;
 
-		if (isDefAndNotNull(opt_content)) {
+		if (opt_content) {
```

Comment:
At some point `opt_content` should be `optContent`, or better still, `optionalContent` (if that's what it is).

Reply:
Maybe we can just do `content`... I don't think we signal "optionality" in general, so we might as well just call it `content` for now and maybe one day this will make sense :D

Reply:
I think I ended up not doing this one just yet...

Reply:
No big deal. Sooner or later we'll get to it if you don't get to it now. (As long as Brian doesn't object when we forward it, but the diff is big enough that he probably won't see it.)